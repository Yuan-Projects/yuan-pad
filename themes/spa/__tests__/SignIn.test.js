import renderer from 'react-test-renderer';
import SignIn from '../src/signIn';
import GuestUser from '../__mocks__/user-guest';
import LangEn from '../__mocks__/site-translate-en';

it('SignIn renders correctly', () => {
  let props1 = {
    user: GuestUser,
    lang: LangEn,
    onCurrentUserUpdated: () =>{}
  };
  const component1 = renderer.create(<SignIn
    {...props1}
  />);
  let tree1 = component1.toJSON();
  expect(tree1).toMatchSnapshot();
});