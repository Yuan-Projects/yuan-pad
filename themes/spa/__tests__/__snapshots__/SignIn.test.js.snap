// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn renders correctly 1`] = `
<div
  className="signIn"
>
  <a
    className="btn btn-default"
    href="#"
    onClick={[Function]}
    role="button"
  >
    Login
  </a>
</div>
`;
